src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -45,23 +45,27 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
                 continue;
             }
 
-            $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
+            $data = $dbForConsole->getDocument(
+                'schedules',
+                $schedule['$id'],
+            )->getAttribute('data', []);
 
+            $delay = $scheduledAt->getTimestamp() - (new \DateTime())->getTimestamp();
 
-            \go(function () use ($queueForFunctions, $schedule, $delay) {
+            \go(function () use ($queueForFunctions, $schedule, $delay, $data, $dbForConsole) {
```

Comment:
`$dbForConsole` isn't used in here, right?

Reply:
nice catch, leftover